Makefile

```diff
@@ -0,0 +1,23 @@
+# Variables
```

Comment:
What is the purpose of this file? Working locally with the repo? I don't see where this is being used

Reply:
Yes, It started to run the `go build` and `go test` without having to add the parameters manually. I also think it's a good practice for developers to run all the same command when building/testing